spec/models/static_members_recruiter_service_spec.rb

```diff
@@ -48,4 +48,56 @@
       expect(mission.members).to include(member)
     end
   end
+
+  context 'when all static slots of all members has a corresponding time slot in missions' do
```

Comment:
`  context 'when all static slots of all members have been assigned to a corresponding time slot in the upcoming missions' do`